.gitignore

```diff
@@ -8,3 +8,4 @@ venv/
 **__pycache__/
 *.pyc
 **db.sqlite3
+media/
```

Comment:
The `media/` directory is ignored. If this directory is used for storing uploaded images, it should not be ignored to comply with the task requirements for handling image uploads.